src/styles/blocks/stars.scss

```
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../../../src/images/star.svg)center no-repeat;
```

There is no need to exit the directory thrice
```suggestion
    background: url(../../images/star.svg) center no-repeat;
```